pages/en/new-design/index.mdx

```diff
@@ -76,17 +76,9 @@ layout: home.hbs
     });
     ```
 
-    ```js displayName="Read a Stream"
-    import { get } from 'node:https';
-
-    const streamContent = [];
-
-    // `get` itself is a stream (Readable Web Stream)
-    get('https://nodejs.org', response => {
-      response.on('data', chunk => streamContent.push(chunk));
-    });
-
-    console.log(streamContent.join());
+    ```js displayName="Fetch a website"
```

Comment:
I think the original purpose of the example was to show basic Stream APIs... Maybe we can update the example to be something like... creating a Readable stream?

Or do you have other suggestions? :eyes:

Reply:
The first example was incorrect - it didn't work if you ran the code.

Reply:
Well, it was still an open PR, but right. I was just messing around with some examples. But the idea for that tab was to be an example of Readable streams. Like a simple example of Readable or Writable streams